### ISSUE TITLE
Allow user to easily add a system message to each message

### ISSUE THREAD

**Issue by joecummings** [pytorch/torchtune#1328]
Currently user has to create a custom message converter class (yuck) or prepend to the user message, which isn't really correct.

**Comment by joecummings** [comment]
In my mind this is a significant gap. Maybe we do something like:

```python

messsage_transform = [AddSystemMessage("You are the best coding assistant around"), AlpacaToMessages()]
```

**Comment by SalmanMohammadi** [comment]
Addressed by #1366
